waterlog/src/Components/Usage.js

```diff
@@ -33,101 +33,81 @@ class Usage extends Component {
-      return (
-        <div>
-          <p className="bold">Cost </p>
-          <DailyCostsReports props={this.props.dailyCost} />
-          <hr id="divide" />
-          <p className="bold">Usage</p>
-          <DailyUsageComponent props={this.props.dailyUsage} />
-        </div>
-      );
+      return <div>
+        <p className="bold">Cost </p>
+        <DailyCostsReports props={this.props}/> 
+        <hr id="divide"/>
+        <p className="bold">Usage</p>
+        <DailyUsageComponent props={this.props.dailyUsage} />
+      </div>
     else if (this.state.display === "monthly")
-      return (
-        <div>
-          <p className="bold">Cost</p>
-          <MonthlyCostsReports props={this.props.monthlyCost} />
-          <hr id="divide" />
-          <p className="bold">Usage</p>
-          <MonthlyUsageComponent props={this.props.monthlyUsage} />
-        </div>
-      );
+      return <div>
+        <p className="bold">Cost</p>
+        <MonthlyCostsReports props={this.props} /> 
+        <hr id="divide"/>
+        <p className="bold">Usage</p>
+        <MonthlyUsageComponent props={this.props.monthlyUsage} />
+      </div>
     else if (this.state.display === "seasonal")
-      return (
-        <div>
-          <p className="bold">Cost</p>
-          <SeasonalCostReports props={this.props.seasonalCost} />
-          <hr id="divide" />
-          <p className="bold">Usage</p>
-          <SeasonalUsageComponent props={this.props.seasonUsage} />
-        </div>
-      );
-    else return <div>Error has occured</div>;
+      return <div>
+        <p className="bold">Cost</p>
+        <SeasonalCostReports props={this.props.seasonalCost} /> 
+        <hr id="divide"/>
+        <p className="bold">Usage</p>
+        <SeasonalUsageComponent props={this.props.seasonUsage} />
+      </div>
+    else
+      return <div>Error has occured</div>
   };
 
   render() {
-    const {
-      dailyError,
-      dailyLoading,
-      monthlyError,
-      monthlyLoading,
-      seasonalError,
-      seasonalLoading
-    } = this.props;
+    const { dailyError, dailyLoading,
+      monthlyError, monthlyLoading,
+      seasonalError, seasonalLoading } = this.props;
     if (dailyError || monthlyError || seasonalError) {
-      return <Error404 />;
+      return <div>Error!</div>;
     }
     if (dailyLoading || monthlyLoading || seasonalLoading) {
-      return <Loader />;
+      return <Loader />
     }
     return (
-      <div>
-        <p className="home-text">
-          <b>WATER USAGE & COST</b>
-        </p>
-        <div className="wastage">
-          <div className="graph-nav tab " id="cost-buttons">
-            <button
-              className={`btn-graph-nav tablinks ${
-                this.state.display === "daily" ? "active" : ""
-              }`}
-              onClick={e => this.openGraph("daily")}
-              id="openByDefault"
-            >
-              Daily
-            </button>
+      <div className="wastage">
```

Comment:
Please leave spaces between your "=". It improves readibility

Reply:
Done